app/_src/gateway/get-started/ai-gateway.md

```diff
@@ -0,0 +1,192 @@
+models are initially supported:
+
+* [Mistral](https://mistral.ai/)
+* [Llama2](https://llama.meta.com/)
+
+See the [AI Proxy plugin configuration](hub/ai-proxy/configuration/) for details on modifying the proxy behavior.
+
+{:.note}
+> **Note:**
+> By default local models are configured on the following endpoint `http://host.docker.internal:11434`
+> which allows {{site.base_gateway}} running in Docker to connect to the host machine. 
+
+### AI Usage Governance
+
+With the growing adoption of AI technologies, organizations are exposed to a set of new risk vectors. In particular,
+organizations are at risk of having sensitive data leaked to AI Providers, exposing them and 
+their customers to data breaches and other security risks. 
+
+Kong's AI Gateway provides additional plugins to aid the organization in controlling AI data and usage. These 
+plugins are used in combination with the AI Proxy plugin, allowing you to build secure and specialized AI 
+experiences for your users.
+
+#### Data Governance
+
+AI Gateway provides the ability to govern outgoing AI prompts via the 
+[AI Prompt Guard](/hub/kong-inc/ai-prompt-guard). This plugin allows the configuration of regular expressions
+following an allow/deny list configuration. Denied prompts result in `4xx` HTTP code responses to clients preventing
+the egress of offending requests.
+
+#### Prompt Engineering
+
+AI systems are built around prompts, and manipulating those prompts is important to successful adoption of the technologies.
+Prompt engineering is the methodology of manipulating the linguistic inputs that guide the AI system. The AI Gateway 
+supports a set of plugins that allow you to create a simplified and enhanced experience by setting default prompts or manipulating
+prompts from clients as they pass through the gateway.
+
+The [AI Prompt Template](/hub/kong-inc/ai-prompt-template) plugin enables administrators to provide pre-configured AI prompts to users. These prompts contain variable 
+placeholders in the format `{{variable}}` which users fill to adapt the template to their specific needs. This functionality 
+prohibits arbitrary prompt injection by sanitizing string inputs to ensure that JSON control characters are escaped. 
+
+The [AI Prompt Decorator](/hub/kong-inc/ai-prompt-decorator) plugin injects an array of `llm/v1/chat` messages at the 
+start or end of a caller’s chat history. This capability allows the caller to create more complex prompts and have more control 
+over how a Large Language Model (LLM) is used when called via {{site.base_gateway}}.
+
+
+#### Request Transformations
+
+Kong's AI Gateway also allows you to use AI technology to augment other API traffic. One example may be routing API responses 
+through an AI language translation prompt before returning it to the client.  Kong's AI Gateway provides two plugins that can be 
+used in conjunction with other upstream API services to weave AI capabilities into API request processing.
+
```

Comment:
Maybe something here about how these plugins can be used without AI Proxy?

Reply:
Added a 1 line comment to that point.